data/sql/updates/pending_db_world/rev_1679851493822859000.sql

```diff
@@ -0,0 +1,23 @@
+-- Mote of Shadow
+DELETE FROM `creature_loot_template` WHERE `Item` = 22577;
+INSERT INTO `creature_loot_template` VALUES
```

Comment:
Always add the column names in here. The columns realigning or changing is unlikely but it's good for cross-core ports, at least.